jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerSessionContext.kt

```diff
@@ -162,135 +170,249 @@ data class CodeModernizerSessionContext(
                         codeTransformSessionId = CodeTransformTelemetryState.instance.getSessionId(),
                         codeTransformMavenBuildCommand = CodeTransformMavenBuildCommand.Mvn,
                         reason = error
                     )
                 } else {
                     shouldTryMvnCommand = false
-                    LOG.warn { "Maven executed successfully" }
+                    LOG.info { "Maven executed successfully" }
                 }
             } catch (e: ProcessNotCreatedException) {
                 val error = "Maven failed to execute as its likely not installed to the PATH"
+                buildlogBuilder.appendLine("mvn failed to execute: $e")
                 CodetransformTelemetry.mvnBuildFailed(
                     codeTransformSessionId = CodeTransformTelemetryState.instance.getSessionId(),
                     codeTransformMavenBuildCommand = CodeTransformMavenBuildCommand.Mvn,
                     reason = error
                 )
-                LOG.warn { error }
+                LOG.error(e) { error }
             } catch (e: Exception) {
+                val error = "Unexpected exception when running mvn"
+                buildlogBuilder.appendLine("mvn failed to execute: $e")
                 CodetransformTelemetry.mvnBuildFailed(
                     codeTransformSessionId = CodeTransformTelemetryState.instance.getSessionId(),
                     codeTransformMavenBuildCommand = CodeTransformMavenBuildCommand.Mvn,
-                    reason = e.message
+                    reason = error
                 )
-                LOG.error(e) { e.message.toString() }
+                LOG.error(e) { error }
             }
         }
 
         // 3. intellij-bundled maven runner
         if (shouldTryMvnCommand) {
-            LOG.warn { "Executing IntelliJ bundled Maven" }
-            val explicitenabled = emptyList<String>()
+            LOG.info { "Executing IntelliJ bundled Maven" }
+            val explicitEnabled = emptyList<String>()
             try {
-                val params = MavenRunnerParameters(
+                val installParams = MavenRunnerParameters(
+                    false,
+                    sourceFolder.absolutePath,
+                    null,
+                    installCommandList,
+                    explicitEnabled,
+                    null
+                )
+
+                val copyParams = MavenRunnerParameters(
                     false,
                     sourceFolder.absolutePath,
                     null,
-                    commandList,
-                    explicitenabled,
+                    copyCommandList,
+                    explicitEnabled,
                     null
                 )
 
                 // Create MavenRunnerParametersMavenRunnerParameters
                 val mvnrunner = MavenRunner.getInstance(project)
                 val transformMvnRunner = TransformMavenRunner(project)
```

Comment:
Lets add logging to the maven runner. We can get maven runner output from the following code in `TransformMavenRunner.kt` 

```
            handler.addProcessListener(object : ProcessAdapter() {
                override fun onTextAvailable(event: ProcessEvent, outputType: Key<*>) {
                    println { "${event.text}" }
                }
                override fun processTerminated(event: ProcessEvent) {
                    println { "------- Process Terminated TEXT -------" }
                    onComplete.exitCode(event.exitCode)
                }
            })
            ```